### PR TITLE
Integration field support

### PR DESCRIPTION
**Pull request by jovitabruziene**
A temporary improvement for integration field - if fragment type is unknown and fragment is not null then return json data as Text

### CODE REVIEW

**Comment by srenault** [comment]
Hey @jovitabruziene, first thanks for your contribution.
We solved the miss of integration field support in the java kit by adding a new fragment type `Raw`. Could you do the same for the csharp kit? (https://github.com/prismicio/java-kit/blob/master/src/main/java/io/prismic/Fragment.java#L1602)

**Comment by srenault** [comment]
Hey @jovitabruziene! Thanks for your PR. I've planned to look at it in the next days.

File: src/prismic/Document.cs [hunk 1]
```diff
@@ -95,6 +95,9 @@ public class Document: WithFragments
 				return fragments;
 			}
 			var type = (string)json["type"];
+			if (type == null)
+                		return fragments;
```
Comment (srenault) [inline src/prismic/Document.cs]: Could you tell me when we encounter this case?

**Comment by srenault** [comment]
Your PR seems nice. Could you add tests?

File: src/prismic/Document.cs (hunk 1 above)
Comment (jovitabruziene) [reply src/prismic/Document.cs]: Seems that this is not encountered, at least I cannot reproduce the situation. So I removed these lines of code in the new commit

**Comment by jovitabruziene** [comment]
@srenault sorry for plenty of commits, it's becayse travis check failed at first, so I had to remove everything and add again. I've removed unnecessary (type == null) test in document.cs and added a json file for raw data type test and a test method.

File: src/prismic.tests/FragmentsTests.cs [hunk 2]
```diff
@@ -110,6 +110,21 @@ public void ShouldParseOEmbed()
 			Assert.IsNull (soundcloud.Obj.Width);
 			Assert.AreEqual (youtube.Obj.Width, 480);
 		}
+		
+		[Test()]
+		public void ShouldAccessRaw()
+		{
+		    var directory = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```
Comment (srenault) [inline src/prismic.tests/FragmentsTests.cs]: We already have an helper function for this: https://github.com/prismicio/csharp-kit/blob/master/src/prismic.tests/Fixtures.cs#L10. Could you use it?

**Comment by srenault** [comment]
Great it's ok for me.

**Comment by jovitabruziene** [comment]
@srenault great! I've updated prismicio to version 1.4.4, but still cannot access method Document.GetRaw(), do you have any idea, why?

**Comment by srenault** [comment]
Weird. Are you sure you're getting the right version. Because NuGet dashboard tells me `0 downloads`
screen shot 2018-05-17 at 12 44 43

**Comment by jovitabruziene** [comment]
Yes. I'm getting error about compatibility with .net core, but this was present in 1.4.3 too.image

**Comment by jovitabruziene** [comment]
@srenault , any news regarding this new release? I cannot access GetRaw method :(

**Comment by srenault** [comment]
It seems you're using .NET Core which is not compatible with this library.

**Comment by jovitabruziene** [comment]
In this file https://github.com/prismicio/csharp-kit/blob/master/src/prismic/WithFragments.cs there is a method public fragments.Raw GetRaw(String field)
And in 1.4.4 release I cannot find it
image

**Comment by srenault** [comment]
Ok I understand. Weird. I can try to republish it.

**Comment by srenault** [comment]
@jovitabruziene I re-published the library and it seems to work fine now. Sorry for the delay.

**Pull request merged**